synapse/daemon.py

```diff
@@ -348,6 +366,7 @@ def _loadDmonConf(self):
         await share.fini()
 
     async def _onTeleSyn(self, link, mesg):
+        assert not self.isfini
```

Comment:
remove since we cannot gaurantee this code will always be present

Reply:
I added a bunch then removed at the end. missed this one. will fix.